source/localizable/feedspec/index.el.html.md.erb

```diff
@@ -529,6 +529,8 @@ To <%= settings.site_name.capitalize %> χρησιμοποιεί ένα προκ
 <size>37/42,37/45</size>
 ~~~
 
+> Αποφύγετε να συμπεριλάβετε το size στον τίτλο του προϊόντος.
```

Comment:
I'd only suggest we keep this consistent with the english version:

```suggestion
> Δεν πρέπει να συμπεριλάβετε το μέγεθος στον τίτλο του προϊόντος με οποιονδήποτε τρόπο.
```

Reply:
Fixup c58d96fc7128bdd9f949ae2d188b28552b676a78